src/main/resources/db/migration/V1_68__create_utilisation_report.sql

```diff
@@ -0,0 +1,307 @@
+-- Function to return daily utilisation stats for a court between two dates
+CREATE OR REPLACE FUNCTION juror_mod.util_report_daily_summary(p_loc_code text, p_start_date date, p_end_date date)
+ RETURNS TABLE(
+	report_date date,
```

Comment:
Is this just serving up the daily report (first tab) - 1 row per date
where's the juror number for the second tab - 1 row per date per juror?
how will this be used for the monthly report? - 1 row per month

Reply:
This report will only provide the daily utilisation stats.

Will need to wait for the design for the per juror view of data. my feeling is we cant do what is done in heritage at the moment, we could provide a separate endpoint/view for one juror or a specific day for the users to drill down on. It would be better from an accessibility viewpoint a well as it reduces the amount of scrolling required.

The monthly report could be done using the daily report over a specific date range and results are aggregated for each month - essentially no more sql dev if it is performant enough.

Reply:
Have spoken to Liam about this - he is happy for us to create an endpoint to focus in on one day - that will return same info as the daily report but have juror column rather than date and then the stats.